docs/release-notes/NuGet-6.8.md

```diff
@@ -0,0 +1,164 @@
+---
+title: NuGet 6.8 Release Notes
+description: Release notes for NuGet 6.8 including new features, bug fixes, and DCRs.
+author: nkolev92
+ms.author: nikolev
+ms.date: 10/30/2023
+ms.topic: conceptual
+---
+
+# NuGet 6.8 Release Notes

```

Comment:
I think we should add a known issues section, for this:

* https://github.com/NuGet/Home/issues/13003

😭

Reply:
We can add these sections like that as needed.